### PR TITLE
#164695527 Add Pagination support for red-flag records

### PR DESCRIPTION
**Pull request by AlvinMugambi**
### What does this PR do?
Adds pagination to the get all red-flag records endpoint

### Description of the task to be completed?
- Improve the get all red-flag records endpoint by including page pagination

### How should you manually test this?

- [x] **Clone the repository**

        $ git clone https://github.com/ann-mukundi/ireporter.git

- [x] **Switch to testing branch**

       $ git checkout ft-redflag-records-pagination-164695527

- [x] **Switch to the ireporter directory**

       $ cd ireporter/

 

- [x] **Create and activate the virtual environment**

      $  virtualenv -p python .venv

      $ source .venv/bin/activate

- [x] **Install project dependencies**

       $ pip install -r api/requirements.txt

- [x] **Run the database migrations**

       $ python api/manage.py migrate

### Running the application

      $ python api/manage.py runserver

- [x] **Signup and activate user**

           POST <your-localhost>/api/auth/signup/

           POST <your-localhost>/api/auth/activate/

- [x] **Login user**

       POST <your-localhost>/api/auth/login/ 

       Use the token provided after login for Authorization and the endpoint
  
- [x] **Create a redflag record**

     

      POST <your-localhost>/api/redflags/

- [x] **Get all redflags first page**

       GET <your-localhost>/api/redflags/?page=1 

- [x] **limit redflag results**

       GET <your-localhost>/api/redflags/?limit=2

### Testing

      $ python api/manage.py test

### Prerequisites

Python, Git,  Virtualenv

### What are the relevant PT stories?

[#164695527](https://www.pivotaltracker.com/story/show/164695527)

### Screenshots
**Success**
Screenshot from 2019-03-20 21-44-41

**Next page**
Screenshot from 2019-03-20 21-56-38

**Limit records to one**
Screenshot from 2019-03-20 21-45-15

### CODE REVIEW

File: api/incidents/views.py [hunk 1]
```diff
@@ -15,6 +16,22 @@
 from django.http.response import Http404
 
 
+# class StandardResultsSetPagination(PageNumberPagination):
```
Comment (Benkimeric) [inline api/incidents/views.py]: Remove this commented code before pushing to GitHub

**Review by Benkimeric — DISMISSED** [review]

File: api/incidents/views.py (hunk 1 above)
Comment (AlvinMugambi) [reply api/incidents/views.py]: Already removed.

**Review by Siffersari — APPROVED** [review]
Good job @AlvinMugambi

**Review by Benkimeric — APPROVED** [review]

**Review by meshack-mbuvi — CHANGES_REQUESTED** [review]
The format for a commit is wrong. Please correct it.

**Comment by AlvinMugambi** [comment]
> The format for a commit is wrong. Please correct it.

Corrected

**Pull request merged**